src/tool/python/strings/pybase.h

```diff
@@ -671,6 +671,63 @@ namespace py
         }
     };
 
+    template <>
+    struct converter<winrt::guid>
+    {
+        static PyObject* convert(winrt::guid value) noexcept
+        {
+            PyObject* valueAsBytes = PyBytes_FromStringAndSize((char*)&value, sizeof(value));
+            PyObject* uuidModule = PyImport_ImportModule("uuid");
```

Comment:
How is the performance of this call in a loop? Is this something that should be lifted our or cached in some way? Could it be static / InitOnceExecuteOnce? Or is it fast enough once loaded that it's not a concern?

Reply:
The way I understand it, `PyImport_ImportModule` will import the module if it is not already imported, otherwise it returns immediately. So in a loop, there should be no performance hit.